<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="13" passed="9">
  <reporter-output>
  </reporter-output>
  <suite name="Page Object test example" duration-ms="410478" started-at="2016-05-04T09:36:58Z" finished-at="2016-05-04T09:43:48Z">
    <groups>
    </groups>
    <test name="Regression 1" duration-ms="410478" started-at="2016-05-04T09:36:58Z" finished-at="2016-05-04T09:43:48Z">
      <class name="com.pp.tests.AccountSummaryTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="setup" is-config="true" duration-ms="14297" started-at="2016-05-04T15:08:47Z" finished-at="2016-05-04T15:09:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyMailMenuOptions()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="verifyMailMenuOptions" duration-ms="41525" started-at="2016-05-04T15:09:01Z" finished-at="2016-05-04T15:09:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMailMenuOptions -->
        <test-method status="PASS" signature="verifyMailMenuSentItems()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="verifyMailMenuSentItems" duration-ms="41878" started-at="2016-05-04T15:09:43Z" finished-at="2016-05-04T15:10:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMailMenuSentItems -->
        <test-method status="PASS" signature="verifyMailMenuComposeMessage()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="verifyMailMenuComposeMessage" duration-ms="61964" started-at="2016-05-04T15:10:25Z" finished-at="2016-05-04T15:11:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMailMenuComposeMessage -->
        <test-method status="FAIL" signature="CheckwithViewProfile()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="CheckwithViewProfile" duration-ms="18443" started-at="2016-05-04T15:11:27Z" finished-at="2016-05-04T15:11:45Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[FAILED THE TEST CASE OF CLICK ON VIEW PROFILE LINK
 CheckwithViewProfile FAILED WHILE CLICKING SIGN OUT BUTTON
 clickOnSignOutButton Element is not clickable at point (1855, 108.5). Other element would receive the click: <div class="jqmOverlay" style="height: 100%; width: 100%; position: fixed; left: 0px; top: 0px; z-index: 2999; opacity: 0.5;"></div>
Command duration or timeout: 61 milliseconds
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'QSIH-00493', ip: '10.145.8.79', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: e28d2fe3-2a19-4722-8ade-2f719b4b8c79]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: FAILED THE TEST CASE OF CLICK ON VIEW PROFILE LINK
 CheckwithViewProfile FAILED WHILE CLICKING SIGN OUT BUTTON
 clickOnSignOutButton Element is not clickable at point (1855, 108.5). Other element would receive the click: <div class="jqmOverlay" style="height: 100%; width: 100%; position: fixed; left: 0px; top: 0px; z-index: 2999; opacity: 0.5;"></div>
Command duration or timeout: 61 milliseconds
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'QSIH-00493', ip: '10.145.8.79', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: e28d2fe3-2a19-4722-8ade-2f719b4b8c79
	at com.pp.tests.AccountSummaryTest.CheckwithViewProfile(AccountSummaryTest.java:289)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckwithViewProfile -->
        <test-method status="FAIL" signature="CheckwithViewProfileAndGettheSlots()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="CheckwithViewProfileAndGettheSlots" duration-ms="10090" started-at="2016-05-04T15:11:45Z" finished-at="2016-05-04T15:11:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Username Lebel is Missing expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Username Lebel is Missing expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.pp.pages.LoginPage.verifyUsernameLabel(LoginPage.java:261)
	at com.pp.tests.AccountSummaryTest.CheckwithViewProfileAndGettheSlots(AccountSummaryTest.java:300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckwithViewProfileAndGettheSlots -->
        <test-method status="FAIL" signature="verifyScheduleMenu()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="verifyScheduleMenu" duration-ms="10152" started-at="2016-05-04T15:11:55Z" finished-at="2016-05-04T15:12:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Username Lebel is Missing expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Username Lebel is Missing expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.pp.pages.LoginPage.verifyUsernameLabel(LoginPage.java:261)
	at com.pp.tests.AccountSummaryTest.verifyScheduleMenu(AccountSummaryTest.java:364)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyScheduleMenu -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.pp.tests.AccountSummaryTest@57829d67]" name="tearDown" is-config="true" duration-ms="308" started-at="2016-05-04T15:12:05Z" finished-at="2016-05-04T15:12:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.pp.tests.AccountSummaryTest -->
      <class name="com.pp.tests.ComposeMessageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.pp.tests.ComposeMessageTest@43a25848]" name="setup" is-config="true" duration-ms="14016" started-at="2016-05-04T15:12:06Z" finished-at="2016-05-04T15:12:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginAndClickonComposeMessage()[pri:0, instance:com.pp.tests.ComposeMessageTest@43a25848]" name="LoginAndClickonComposeMessage" duration-ms="37809" started-at="2016-05-04T15:12:20Z" finished-at="2016-05-04T15:12:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginAndClickonComposeMessage -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.pp.tests.ComposeMessageTest@43a25848]" name="tearDown" is-config="true" duration-ms="322" started-at="2016-05-04T15:12:58Z" finished-at="2016-05-04T15:12:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.pp.tests.ComposeMessageTest -->
      <class name="com.pp.tests.GenericAppointmentRequestTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.pp.tests.GenericAppointmentRequestTest@30c7da1e]" name="setup" is-config="true" duration-ms="13242" started-at="2016-05-04T15:12:58Z" finished-at="2016-05-04T15:13:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="LoginAndRequestForAppointment()[pri:0, instance:com.pp.tests.GenericAppointmentRequestTest@30c7da1e]" name="LoginAndRequestForAppointment" duration-ms="36885" started-at="2016-05-04T15:13:11Z" finished-at="2016-05-04T15:13:48Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[FAILED THE TEST CASE OF CLICK ON LOGIN BUTTON
 clickOnLoginWithValidCredentials  FAILED WHILE SELECTING THE PRACTICE DROPDOWN
 selectPracticeDropdownUnable to locate element: {"method":"id","selector":"ctl00_ContentPlaceHolder1_PracticePersonSelector1_ddlPersons"}
Command duration or timeout: 10.15 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'QSIH-00493', ip: '10.145.8.79', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 687c9cd3-66da-4f16-906f-321506f076b2
*** Element info: {Using=id, value=ctl00_ContentPlaceHolder1_PracticePersonSelector1_ddlPersons}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: FAILED THE TEST CASE OF CLICK ON LOGIN BUTTON
 clickOnLoginWithValidCredentials  FAILED WHILE SELECTING THE PRACTICE DROPDOWN
 selectPracticeDropdownUnable to locate element: {"method":"id","selector":"ctl00_ContentPlaceHolder1_PracticePersonSelector1_ddlPersons"}
Command duration or timeout: 10.15 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'QSIH-00493', ip: '10.145.8.79', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 687c9cd3-66da-4f16-906f-321506f076b2
*** Element info: {Using=id, value=ctl00_ContentPlaceHolder1_PracticePersonSelector1_ddlPersons}
	at com.pp.tests.GenericAppointmentRequestTest.LoginAndRequestForAppointment(GenericAppointmentRequestTest.java:138)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginAndRequestForAppointment -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.pp.tests.GenericAppointmentRequestTest@30c7da1e]" name="tearDown" is-config="true" duration-ms="312" started-at="2016-05-04T15:13:48Z" finished-at="2016-05-04T15:13:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.pp.tests.GenericAppointmentRequestTest -->
      <class name="com.pp.tests.LoginPageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="setup" is-config="true" duration-ms="20229" started-at="2016-05-04T15:06:58Z" finished-at="2016-05-04T15:07:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyIamNewHereTestCase()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="verifyIamNewHereTestCase" duration-ms="10088" started-at="2016-05-04T15:07:18Z" finished-at="2016-05-04T15:07:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIamNewHereTestCase -->
        <test-method status="PASS" signature="verifyAllReadyMembersElements()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="verifyAllReadyMembersElements" duration-ms="73" started-at="2016-05-04T15:07:28Z" finished-at="2016-05-04T15:07:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAllReadyMembersElements -->
        <test-method status="PASS" signature="clickOnLoginWithInvalidUsernameAndPassword()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="clickOnLoginWithInvalidUsernameAndPassword" duration-ms="8624" started-at="2016-05-04T15:07:28Z" finished-at="2016-05-04T15:07:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnLoginWithInvalidUsernameAndPassword -->
        <test-method status="PASS" signature="clickOnNeedHelpWithyourUsernamePassLink()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="clickOnNeedHelpWithyourUsernamePassLink" duration-ms="11140" started-at="2016-05-04T15:07:37Z" finished-at="2016-05-04T15:07:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnNeedHelpWithyourUsernamePassLink -->
        <test-method status="PASS" signature="clickOnLoginWithValidCredentials()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="clickOnLoginWithValidCredentials" duration-ms="58616" started-at="2016-05-04T15:07:48Z" finished-at="2016-05-04T15:08:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnLoginWithValidCredentials -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.pp.tests.LoginPageTest@19dfb72a]" name="tearDown" is-config="true" duration-ms="443" started-at="2016-05-04T15:08:47Z" finished-at="2016-05-04T15:08:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.pp.tests.LoginPageTest -->
    </test> <!-- Regression 1 -->
  </suite> <!-- Page Object test example -->
</testng-results>
